gql/transport/requests.py

```diff
@@ -275,6 +274,152 @@ def raise_response_error(resp: requests.Response, reason: str):
             extensions=result.get("extensions"),
         )
 
+    def execute_batch(  # type: ignore
+        self,
+        reqs: List[GraphQLRequest],
+        timeout: Optional[int] = None,
+        extra_args: Dict[str, Any] = None,
```

Comment:
should be `Optional[Dict[str, Any]]`. I noticed that the same mistake was already present in the execute method and that could be corrected also.

Reply:
Of course! I've just copied that from the old code